ec2-windows-blueprint.yaml

```diff
@@ -0,0 +1,93 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+description: >
+  This blueprint defines a Windows Server 2012 R2 VM on AWS EC2 and an elastic IP
+  and a security group are also created and associated with the VM all using Cloudify's AWS plugin.
+  In addition, a simple web server is created and started using Cloudify's script plugin.
+  Note: This blueprint can also work with a Windows Server 2008 R2 VM on AWS EC2.
+        To do so omit the single quotes in the vm user_data under the node_templates section below.
+
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.4rc1/types.yaml
+  - http://www.getcloudify.org/spec/aws-plugin/1.4/plugin.yaml
+
+inputs:
+  webserver_port:
+    description: The HTTP web server port
+    default: 8080
+  agent_user:
+    description: User name used to access the AWS EC2 VM
+    default: Administrator
+  image_id:
+    description: AWS EC2 image id to use for the server
+  instance_type:
+    description: AWS EC2 instance type to use for the server
+    default: m3.medium
+
+node_templates:
```

Comment:
Good call. aside from using an empty line as a separator between node templates. should there be an empty line to separate between inputs, node_templates and outputs? had a look at other yaml files and its not entirely clear based on that.